### ISSUE TITLE
Hyperdrive Write Modifies mtime so resume doesn't act as expected

### ISSUE THREAD

**Issue by joehand** [juliangruber/hyperdrive-import-files#12]
When the files are written to hyperdrive, the mtime get updated for most (not all) files. This causes the resume to almost always think files are updated. 

This is more of a hyperdrive issue, just leaving a reminder here.

You can do a quick check adding a `fs.stat` in the `consumeFile` [pump function](https://github.com/juliangruber/hyperdrive-import-files/blob/master/index.js#L63):

```js
pump(rs, ws, err => {
  if (err) return cb(err)
  entry = entries[hyperPath] = entry || {}
  entry.length = stat.size
  entry.mtime = stat.mtime.getTime()
  fs.stat(file, function (err, stat) {
    if (entry.mtime !== stat.mtime.getTime()) console.log('mtimes not equal') // prints for most files
    status.emit('file imported', {
      path: file,
      mode
    })
    cb()
  })
})
```

**Comment by joehand** [comment]
Using your hyperdrive PR, the first file the mtime gets through okay, but subsequent files the mtime is wrong. 

I [tested multiple files](https://gist.github.com/joehand/dec789618840c97646306fd795e2bf0b) in hyperdrive and it passes fine. So I think it is a bug in this module.

**Comment by joehand** [comment]
Hmm, looks like if you run hyperdrive-import-files multiple times the mtime gets changed each time for all the files.

**Comment by juliangruber** [comment]
found the error: this hard coded offset of the zero-th storage unit potentially leads to the wrong random-access-file instance, so we're not setting the mtimes on the files that we want to

**Comment by juliangruber** [comment]
hopefully fixed by https://github.com/mafintosh/hyperdrive/pull/92/commits/e839eb20f4b8cf4a69617d46cfd43b7715a0258f

**Comment by joehand** [comment]
@juliangruber your mtime PR was merged & released hyperdrive 7.3.0 so should be fixed.
